landoui/server.py

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think you should rename this module.  I named this module `app.py` in lando-api because the module's primary concern is building Flask application objects for other functions to use, such as test suites and wsgi app runners.